CHANGELOG.md

```diff
@@ -49,7 +51,10 @@ Ref: https://keepachangelog.com/en/1.0.0/
 ### Bug Fixes
 
 * (client) [#10226](https://github.com/cosmos/cosmos-sdk/pull/10226) Fix --home flag parsing. 
-* [\#10394](https://github.com/cosmos/cosmos-sdk/issues/10394) Fixes issue related to grpc-gateway of account balance by ibc-denom.
+
+### Client Breaking Changes
+
+* [\#10394](https://github.com/cosmos/cosmos-sdk/issues/10394) Fixes issue related to grpc-gateway of account balance by ibc-denom. Please use `/cosmos/bank/v1beta1/balances/{address}/by_denom?denom={DENOM or IBC-DENOM}` for querying balances by denom.
```

Comment:
there are a few other changes, do we want to include them as well?

Reply:
Aren't they documentedd L41-L53? Or did we forget some?

In this PR, I changed this item from "Bug Fixes" to "Client Breaking Changes"

Reply:
oh yea missed them, sorry